### PR TITLE
[HOPSWORKS-1377][featurestore][queryplanner] composite key support

### PR DESCRIPTION
**Pull request by Limmen**

### CODE REVIEW

File: hops/featurestore.py [hunk 1]
```diff
@@ -2052,7 +2053,7 @@ def import_featuregroup_redshift(storage_connector, query, featuregroup, primary
     fs_utils._log("Feature group imported successfully")
 
 
-def import_featuregroup_s3(storage_connector, path, featuregroup, primary_key=None, description="", featurestore=None,
+def import_featuregroup_s3(storage_connector, path, featuregroup, primary_keys=None, description="", featurestore=None,
```
Comment (moritzmeister) [inline hops/featurestore.py]: In the other methods you use `[]` as default for `primary_keys`, any reason why not here?

File: hops/featurestore.py [hunk 2]
```diff
@@ -2068,7 +2069,7 @@ def import_featuregroup_s3(storage_connector, path, featuregroup, primary_key=No
     >>> featurestore.import_featuregroup_s3(my_s3_connector_name, s3_path, featuregroup_name,
     >>>                                  data_format=s3_bucket_data_format)
     >>> # You can also be explicitly specify featuregroup metadata and what statistics to compute:
-    >>> featurestore.import_featuregroup_s3(my_s3_connector_name, s3_path, featuregroup_name, primary_key="id",
+    >>> featurestore.import_featuregroup_s3(my_s3_connector_name, s3_path, featuregroup_name, primary_keys="id",
```
Comment (moritzmeister) [inline hops/featurestore.py]: Shouldn't it be list here too?

Comment (Limmen) [reply hops/featurestore.py]: thx, fixed

File: hops/featurestore.py (hunk 1 above)
Comment (Limmen) [reply hops/featurestore.py]: thx, fixed

File: hops/featurestore.py (hunk 2 above)
Comment (Limmen) [reply hops/featurestore.py]: thx, fixed

**Pull request closed without merging**